gulp/tasks/copy.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var changed = require( 'gulp-changed' );
+var replace = require( 'gulp-replace' );
+var config = require( '../config' ).copy;
```

Comment:
name `var configCopy` or something to differentiate from `var config = require( '../config' );`